server_installation/topics/cache/eviction.adoc

```diff
@@ -32,4 +35,24 @@ subsystem, which looks similar to this:
 ----
 
 To limit or expand the number of allowed entries simply add or edit the `object` element or the `expiration` element of particular cache
-configuration. 
\ No newline at end of file
+configuration.
+
+In addition, there are also separate caches `sessions`, `clientSessions`, `offlineSessions`, `offlineClientSessions`,
+`loginFailures` and `actionTokens`. These caches are distributed in cluster environment and they are unbounded in size by default.
+If they are bounded, it would then be possible that some sessions will be lost. Expired sessions are cleared internally
+by {project_name} itself to avoid growing the size of these caches
+without limit. If you see memory issues due the big amount of sessions, you can try to:
+
+* Increase the size of cluster (more nodes in cluster means that sessions are spread more equally among nodes)
+
+* Increase the memory for {project_name} server process
+
+* Lower the amount of owners to ensure that caches are saved just on single place. See <<_replication>> for more details
```

Comment:
* Decrease the number of owners to ensure that caches are saved in one single place.